src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.ts

```diff
@@ -427,9 +451,24 @@ const stakeRequest = async (opts: EdgeGuiPluginOptions, request: ChangeQuoteRequ
         allocationType: 'fee',
```

Comment:
Shouldn't we rename `'fee'` to `'networkFee'` sense we're expanding the type to with other fee types?

Reply:
cleanup commit to change to `networkFee`